system/lib/compiler-rt/extras.c

```diff
@@ -0,0 +1,22 @@
+// Supprot functions for emscripten setjmp/longjmp and excecption handling
+// support.
+// See: https://llvm.org/doxygen/WebAssemblyLowerEmscriptenEHSjLj_8cpp.html
+
+int __THREW__;
+int __threwValue;
+int __tempRet0;
+
+void setThrew(int threw, int value) {
+  if (__THREW__ == 0) {
+    __THREW__ = threw;
+    __threwValue = value;
+  }
+}
+
+void setTempRet0(int value) {
+  __tempRet0 = value;
+}
+
+void getTempRet0(int value) {
+  __tempRet0 = value;
```

Comment:
This is the same as setTempRet0. Should be something like
```
int getTempRet0() {
  return __tempRet0;
}
```

By the way, do we need this? I don't think we generated `getTempRet0` in the LLVM pass or needed it.

Reply:
We've generated or needed it before for runtime compatibility reasons... I thought we did this in binaryen in wasm-emscripten.cpp, but I don't see it anywhere. Not even in s2wasm from back when that still existed.
So maybe we don't need this? Somehow?